.github/workflows/db-release.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow updates the database and creates a release of it everyday
+
+name: Release Workflow
+on:
+  schedule:
+  - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'src/tech_list.csv'
+
+jobs:
+  db-release:
+    name: "Database Release"
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Install requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install --no-deps -r requirements.txt
+          pip install --no-deps .
+
+      - uses: "robinraju/release-downloader@v1.2"
+        continue-on-error: true
+        with:
+          repository: "Cyberwatch/HashThePlanet"
+          latest: true
+          fileName: "hashtheplanet.db"
+          tarBall: false
+          zipBall: false
+          out-file-path: "dist/"
+
+      - name: "Build"
```

Comment:
You could find a better name for this step, like "Compute database" or "Update database"